include/uk/plat/lcpu.h

```diff
@@ -41,6 +41,27 @@
 extern "C" {
 #endif
 
+#define UKPLAT_PER_LCPU_VAR_DEFINE(var_type, var_name) \
```

Comment:
I am wondering if we can shorten the macro names a bit, so that we do not reach the 80 char limit too quickly:
For example:
`UKPLAT_PER_LCPU_VAR_DEFINE()` -> `UKPLAT_PER_LCPU_DEFINE()`
`UKPLAT_PER_LCPU_FIRST_MULTI_DIM_VAR_DEFINE` -> `UKPLAT_PER_LCPU_ARRAY_DEFINE`

Reply:
+1 from me, I had the same thought on that.

Reply:
Ack, I'll update to the proposed shorter naming. Thank you.